app/packages/operators/src/StartupOperatorExecutor.tsx

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from "react";
+import { executeStartupOperators } from "./operators";
+
+let startupOperatorsExecuted = false;
+
+export default function StartupOperatorExecutor() {
```

Comment:
This seems like a workaround to the actual problem. How can we make the following possible regardless of where or when it happens:

```ts
const queue = getInvocationRequestQueue();
queue.add(new InvocationRequest());
```

Reply:
> There are alternate ways to go about solving this by notifying the subscriber on add or subscriber checking the queue when adding itself as a subscriber to the queue.

This

Reply:
Its important that `executeOperator()` is essentially possible from any state, thats why we introduced the queue in the first place.